db/migrate/20110317144932_add_session_table.rb

```diff
@@ -1,7 +1,11 @@
 class AddSessionTable < ActiveRecord::Migration[4.2]
 
 	class Event < ApplicationRecord
-		serialize :info
+		if ActiveRecord::VERSION::MAJOR >= 7 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
we're tab indented here for some reason, ignore the formatting it looks normal in the IDE, file was orinigally tabs so not going to change that